bliss/models/decoder.py

```diff
@@ -223,21 +222,26 @@ def _render_ptiles(self, tile_catalog: TileCatalog) -> Tensor:
         )
 
         # draw stars and galaxies
-        stars = self.star_tile_decoder(locs, star_fluxes, star_bools)
+        centered_stars = self.star_tile_decoder.forward(star_fluxes, star_bools)
+        stars = self.tiler.forward(locs, centered_stars)
         galaxies = torch.zeros(img_shape, device=locs.device)
         lensed_galaxies = torch.zeros(img_shape, device=locs.device)
 
         if self.galaxy_tile_decoder is not None:
-            galaxies = self.galaxy_tile_decoder(locs, tile_catalog["galaxy_params"], galaxy_bools)
+            centered_galaxies = self.galaxy_tile_decoder.forward(
```

Comment:
you changed the class name to just `GalaxyDecoder` but you still call the object `galaxy_tile_decoder`.

Reply:
Good catch. Now we have `galaxy_decoder.galaxy_decoder` in a few places, which makes me question whether we need a separate `GalaxyDecoder` class at all. I'll raise a refactoring issue